calcom/cal.com
# Hide other attendees on event types with seats

## What does this PR do?



This PR allows hiding attendee information from other attendees on event types with seats. The .ics file that is generated for attendees will also not contain other attendee information as well.

![image](https://user-images.githubusercontent.com/65426560/193118862-e03f0a2b-4606-4769-a5ec-a7c7ee0370b5.png)


Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Edit an event type to contain seats and check the box to hide attendee information
- Book at least two seats in one time slot
    - In the second email you should see the .ics file for the organizer contains two attendees while the .ics file to the attendee should only contain their information and the organizer'
- Now disable hide attendee information
- When booking seats in the same time slot the attendee information should stack 	

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
